### PR TITLE
ROC-2892: Sync settlement agreement PDF with prototype

### PR DESCRIPTION
**Pull request by geek-soft**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-2892

### Change description ###

This PR updates settlement agreement PDF to match prototype agreed with Marc.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/staff/templates/document/settlementAgreement.html [hunk 1]
```diff
@@ -174,20 +256,40 @@ <h2>Settlement agreement</h2>
...
-    <strong>Settlement agreement</strong>
+    <strong>What this agreement means</strong>
   </div>
   <div class="section-body">
     <table>
       <tr>
-        <td>Agreement:</td>
         <td>
-          {{ acceptedOffer }}
+          <p>This agreement settles the claim made by Jan Clark against Mary Richards. This includes all money owed in the claim, for example court fees, expenses or interest.</p>
```
Comment (timja-kainos) [inline src/main/resources/staff/templates/document/settlementAgreement.html]: jan clark and mary richards ae?

Comment (geek-soft) [reply src/main/resources/staff/templates/document/settlementAgreement.html]: Ups, fast and furious changes, good spot.

Comment (geek-soft) [reply src/main/resources/staff/templates/document/settlementAgreement.html]: Done

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
     private final Address address;
     private final boolean addressAmended;
     private final Address correspondenceAddress;
+    private final String mobilePhone;
```
Comment (robertparkinson) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: isn't this just supposed to be phoneNumber now? we don't ask for a mobile, just a contact number

Comment (geek-soft) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/PartyDetailsContent.java]: Good point, we should rename it at some point but doing it now would trigger too many changes, both in frontend and backend.

**Pull request merged**